### ISSUE TITLE
Application Rendering Error: edge 20

### ISSUE THREAD

**Issue by estelle** [mozilla/kuma#5993]
**Summary**
Getting 
Application Rendering Error
An unhandled error occured in the application. We aplogize for the inconvenience!
<button> reload page
In Edge 20 (trying to update to 44) on Windows 10.

**Steps To Reproduce (STR)**
Visit any page on developer.mozilla.org with Edge 20 on windows 10

**Actual behavior**
Home page loaded with error on top.
Clicking on "web technologies" I go to developer.mozilla.org/en-US/docs/Web/ but the page is blank other than the error.

**Expected behavior**
The page to load without error

**Additional context**
Filing this bug from my mac, new to the pc, so no screen shot know-how. yet. And not logged into anything on that device yet. sorry no screen shot attached.

**Comment by peterbe** [comment]
That's super useful!! What does it say in the Web Console (or whatever the equivalent is called)?

**Comment by peterbe** [comment]
On behalf of Estelle, I'm attaching the screenshot from her:
![bugreport](https://user-images.githubusercontent.com/26739/66923277-7f679980-eff6-11e9-9193-42d45b059247.png)

**Comment by peterbe** [comment]
Note-to-self; https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Advanced_styling_for_HTML_forms

@estelle Sorry for pestering you with questions. There's an `[object Error]` there. We can see a "brief" part of it. Can you try to expand that part and take a screenshot of that?

Also, I carefully checked we have no mention anywhere in our code or our compiled code of a `ToString` function, variable, property, or anything. I suspect this is a bug in Edge 20 itself that might only happen when it tries to deal with errors.

**Comment by peterbe** [comment]
Actually, I really think the relevant error is a `TypeError` about `fetch` not existing. I tried that URL in my Firefox 38 and the (first) error you get is:
```
TypeError: t.insertAdjacentElement is not a function
```

But I also attempted that same page, locally, with Firefox 38, using the branch from [my PR](https://github.com/mozilla/kuma/pull/5996) and it does fix(*) the page. 
<img width="1616" alt="Screen Shot 2019-10-16 at 9 36 13 AM" src="https://user-images.githubusercontent.com/26739/66924694-12a1ce80-eff9-11e9-9f4b-8828c963eb92.png">

(*) The CSS is still busted but at least I can read it.
